src/renderers/WebGLRenderer.js

```diff
@@ -629,7 +629,12 @@ class WebGLRenderer {
 			}
 
 			if ( depth ) bits |= _gl.DEPTH_BUFFER_BIT;
-			if ( stencil ) bits |= _gl.STENCIL_BUFFER_BIT;
+			if ( stencil ) {
+
+				bits |= _gl.STENCIL_BUFFER_BIT;
+				this.state.buffers.stencil.setMask( 0xff );
```

Comment:
Thanks for the example, that helped to clear things up.

Do you mind using the same default mask value like in `WebGLState` `0xffffffff`? I've tested it and the clear also works with this value.

Reply:
Makes sense, updated.